includes/class-cron.php

```
@@ -139,30 +139,43 @@ private function schedule_page_cron_jobs(): void {
+		foreach ( $exclude_urls as $exclude_url ) {
+			$exclude_url = rtrim( $exclude_url, '/' );
+
+			if ( 0 !== strpos( $exclude_url, 'http' ) ) {
+				$exclude_url = home_url( $exclude_url );
+			}
+
+			if ( false !== strpos( $exclude_url, '(.*)' ) ) {
+				$prefix_excludes[] = str_replace( '(.*)', '', $exclude_url );
+			} else {
+				$exact_excludes[] = $exclude_url;
+			}
```

PHPMD (2.15.0)

[error] The method schedule_page_cron_jobs uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)